webapp/views.py

```diff
@@ -921,4 +922,4 @@ def marketo_submit():
     if return_url:
         return flask.redirect(return_url)
 
-    return flask.jsonify({"message": "Form submitted."})
+    return flask.redirect("/thank-you")
```

Comment:
I know this was my suggestion, but it was just illustrative. `/thank-you` is a 404 :cry:

Reply:
Sorry, created a thank you page as a separate commit!